### ISSUE TITLE
Rethink that red background highlight

### ISSUE THREAD

**Issue by Infernio** [wrye-bash/wrye-bash#656]
Right now we use it for:
- `Deactivate`-tagged plugins that are active.
- Two plugins that share the same mtime (Last Modified timestamp).

Problems with this:
1. The first one isn't nearly as bad as the second one - if you're not using a BP, there is no point in ever worrying about it and you might as well just set the color to white in the settings. Plus the BP already gives you a prompt to deactivate all `Deactivate`-tagged ones. And the Plugin Checker warns too. *And* we mark the plugin in italics.
1. The highlighting makes the entry pretty much unreadable, especially if you are visually impaired or colorblind.
1. The fact that *these two* problems are the ones that get this highlighting is odd. There are some much more serious issues that WB does not highlight this way (active plugin with missing masters, Skyrim LE BSA on Skyrim SE, etc. - those cause crashes, but we don't do any kind of weird highlight there).

As a [result](https://reddit.com/r/oblivionmods/comments/xzx0rt/what_do_these_red_boxes_mean_and_how_do_i_fix_it/), people [keep](https://cdn.discordapp.com/attachments/537708965942001685/1055148817806282762/image.png) getting [confused](https://user-images.githubusercontent.com/2845240/208955043-f65412d8-062c-479f-b051-adea3e96a29e.png) by this.

So my proposal is:
- Remove the highlighting in the first case (redundant as mentioned above, plus makes it harder to figure out at a glance what problems your load order has).
- Tone down the color in the second case to make it more readable (quick draft below).

![image](https://user-images.githubusercontent.com/2845240/208954337-b985bb8d-3d77-4d53-8494-a38201b20a07.png)

*(lighter color is inactive mtime conflict, darker color is active mtime conflict)*

**Comment by Utumno** [comment]
Good solution 👍 people [keep getting confused](https://discord.com/channels/537706885965676554/537710082755133460/1060093606054080583) over the "Two plugins share the same mtime." color also - could be worth it to actually redate the mods on version changed but laterzzz
